descopesdk/src/main/java/com/descope/internal/http/DescopeClient.kt

```diff
@@ -332,7 +334,7 @@ internal class DescopeClient(private val config: DescopeConfig) : HttpClient(con
     override val defaultHeaders: Map<String, String> = mapOf(
         "Authorization" to "Bearer ${config.projectId}",
         "x-descope-sdk-name" to "android",
```

Comment:
```suggestion
        "x-descope-sdk-name" to Descope.name,
```

Reply:
we use lowercase platform name in every SDK, so let's leave it like this (in swift it's `swift` not `DescopeKit`)

the name constant can be used in other place, like logs etc